docs/targets/s3.md

```diff
@@ -0,0 +1,67 @@
+# Event Target for AWS S3
+
+This event Target receives [CloudEvents][ce] over HTTP and invokes an AWS S3 endpoint.
+
+## Prerequisites
+
+1. AWS API key and secret
+
+1. ARN for the S3 bucket to store the event
+
+Consult the [Secrets](../guides/secrets.md) guide for more information about
+how to add the AWS API specific secrets.
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a Target of type `AWS S3`.
+
+![Adding an S3 Target](../images/aws-targets/aws-s3-bridge-create-1.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **AWS Secret**: Reference a [TriggerMesh secret][tm-secret] containing an AWS API key and Secret as discussed in the prerequisites
+* **AWS ARN**: The ARN that points to the AWS S3 topic
+
+There is an optional toggle flag indicating if the full CloudEvent should be sent
+to S3 bucket. By default, this is disabled which means the event payload
+will be sent only.
+
+![AWS S3 Target form](../images/aws-targets/aws-s3-bridge-create-2.png)
+
+After clicking the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
+
+![Bridge overview](../images/aws-targets/aws-s3-bridge-create-3.png)
+
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with the AWS S3 Target was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
```

Comment:
nit picky - would we prefer an image that shows the icons in the table? I know we use this image in other locations, if we do change it here, we would then need to add more images. Maybe having no icons is reusable?

Reply:
Good question. Think at this time having the single image for reuse should be okay as the only difference is the bridge name and possibly target icon.